nntrainer/src/elem_add_layer.cpp

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (C) 2020 Parichay Kapoor <pk.kapoor@samsung.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0-only
+ *
+ * @file	elem_add_layer.cpp
+ * @date	30 July 2020
+ * @see		https://github.com/nnstreamer/nntrainer
+ * @author	Parichay Kapoor <pk.kapoor@samsung.com>
+ * @bug		No known bugs except for NYI items
+ * @brief	This is Element-Wise Layer Class for Neural Network
+ *
+ */
+
+#include <elem_add_layer.h>
+#include <layer.h>
+#include <nntrainer_error.h>
+#include <nntrainer_log.h>
+#include <parse_util.h>
+#include <util_func.h>
+
+namespace nntrainer {
+
+int ElemAddLayer::initialize(bool last) {
```

Comment:
Add validation for `num_inputs`?

Reply:
can you elaborate on what kind of validation.
`num_inputs` is Initialized with valid value and setProperty() ensures that the value remains valid.

Reply:
if that's the case I think it is not needed.